client/verta/verta/client.py

```diff
@@ -780,6 +780,7 @@ def get_or_create_registered_model(
         id=None,
         task_type=None,
         data_type=None,
+        pii=None,
```

Comment:
```suggestion
        pii=False,
```

See my other comment below, on the docstring